src/details/ArborX_DetailsUtils.hpp

```diff
@@ -23,6 +24,16 @@ namespace ArborX
 
 namespace Details
 {
+
+template <class T>
+struct remove_cvref
+{
+  typedef std::remove_cv_t<std::remove_reference_t<T>> type;
+};
+
+template <class T>
+using remove_cvref_t = typename remove_cvref<T>::type;
```

Comment:
That's exactly what `std::decay_t` is doing (if `T` is not a function or an array). https://en.cppreference.com/w/cpp/types/decay

Reply:
I know about `std::decay`.  Are you asking me to change?

Reply:
Yes, I don't see a reason to define a separate type trait if there is one in `<type_traits>` that does the same (unless you have a reason why that is not good enough).